crypto/ec/ecp_nistz256.c

```diff
@@ -1445,6 +1446,131 @@ static int ecp_nistz256_inv_mod_ord(const EC_GROUP *group, BIGNUM *r,
 # define ecp_nistz256_inv_mod_ord NULL
 #endif
 
+// Full init has to muck about with too much stuff
+// Plan: write piece by piece, add more precomputed data later
+static int ecp_nistz256group_full_init(EC_GROUP *group, const unsigned char *params) {
+    BN_CTX *ctx = NULL;
+    BN_MONT_CTX *mont = NULL;
+    int param_len = 32;
+    int seed_len = 20;
+    int ok = 0;
+    BIGNUM *p = NULL, *a = NULL, *b = NULL, *x = NULL, *y = NULL, *one = NULL, *order =
+        NULL;
+    EC_POINT *P = NULL;
+
+    if ((ctx = BN_CTX_new_ex(NULL)) == NULL) {
+        ERR_raise(ERR_LIB_EC, ERR_R_MALLOC_FAILURE);
+        goto err;
```

Comment:
return 0;

Reply:
And I also think we don't need ERR_raise here

Reply:
pointless going to err here, you havent alloced anything at this point